### DISCUSSION TITLE
Container name as UUID

### DISCUSSION THREAD

**Discussion by justinwalz** [aquasecurity/trivy-operator#410] [Help & Support]
Hi. Thanks for the work on this project. Question about the container names. Why are the init container names UUIDs? https://github.com/aquasecurity/trivy-operator/blob/021ffe8e88d6e4f3d6d4ca932c6c8311c2bc3bb1/pkg/plugins/trivy/plugin.go#L1107

For systems that collect metrics/logs, this would inherently spike cardinality as there are now potentially infinite values for the label `container` or `containerName`. 

It seems like they could just be `copy-binary` or `db`, without much issue. 

Let me know what you all think. Best.

**Comment by chen-keinan** [comment]
@justinwalz thank you for this input I assume it can be improved.
 
 Can you please open an issue for it ?
